napari/layers/image/image.py

```diff
@@ -517,12 +524,66 @@ def interpolation(self, interpolation):
         self.events.interpolation(value=self._interpolation[self._ndisplay])
 
     @property
-    def experimental_slicing_plane(self):
-        return self._experimental_slicing_plane
+    def depiction(self):
+        """The current 3D depiction mode.
+
+        Selects a preset depiction mode in vispy
+            * volume: images are rendered as 3D volumes.
+            * plane: images are rendered as 2D planes embedded in 3D.
+                plane position, normal, and thickness are attributes of
+                layer.plane which can be modified directly.
+        """
+        return str(self._depiction)
+
+    @depiction.setter
+    def depiction(self, depiction: Union[str, VolumeDepiction]):
+        """Set the current 3D depiction mode."""
+        self._depiction = VolumeDepiction(depiction)
+        self._update_plane_callbacks()
+        self.events.depiction()
+
+    def _reset_plane_parameters(self):
+        """Set plane attributes to something valid."""
+        self.plane.position = np.array(self.data.shape) / 2
+        self.plane.normal = (1, 0, 0)
+
+    # The following callbacks are added as partials otherwise they are
+    # considered as methods on the class, this is similar to this problem:
+    # https://stackoverflow.com/questions/40338652/how-to-define-enum-values-that-are-functions
+    _plane_drag_callback = partial(move_plane_along_normal)
+    _plane_double_click_callback = partial(set_plane_position)
+
+    def _update_plane_callbacks(self):
+        """Connect or disconnect plane callbacks as appropriate."""
+        plane_drag_callback_connected = (
+            self._plane_drag_callback in self.mouse_drag_callbacks
+        )
+        double_click_callback_connected = (
+            self._plane_double_click_callback
+            in self.mouse_double_click_callbacks
+        )
+        if self.depiction == VolumeDepiction.VOLUME:
+            if plane_drag_callback_connected:
+                self.mouse_drag_callbacks.remove(self._plane_drag_callback)
```

Comment:
Would it be possible to directly attach/remove `move_plane_along_normal()` here rather than pass it through `self._plane_drag_callback`? Doing so would eliminate the adding the function as an attribute above.